ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/EvmTool.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+package org.hyperledger.besu.evmtool;
+
+import static picocli.CommandLine.defaultExceptionHandler;
+
+import picocli.CommandLine;
+
+public final class EvmTool {
```

Comment:
Why do we need a separate main class ? I thought it was a classical subcommand.

Reply:
For now this is not meant for general consumption, this tool doesn't get added to the distribution.  If that changes it will be moved in or some other policy set in place.

Reply:
Sounds good.